tools/scripts/generic_variables.mk

```diff
@@ -34,6 +34,9 @@ GIT_VERSION := $(subst -0-g,-,$(GIT_VERSION))
 CFLAGS += -DNO_OS_VERSION=$(GIT_VERSION) \
 		-DNO_OS_PROJECT=$(notdir $(PROJECT))
 
+#Variables for compiling flags in generic.mk
+CFLAGS_MFLOAT_TYPE 	:= hard
```

Comment:
why not `?=` . I suppose the purpose of this variable is to be able to customize the value right?

Reply:
`?=` might make more sense, will add the changes